### PR TITLE
Added content for load authorization keys feature

### PR DESCRIPTION
**Pull request by GloreenS**
### Related links

Card #189 

### Changes

New topic for Listing authorization keys

### CODE REVIEW

File: source/docs/casper/dapp-dev-guide/load-auth-keys.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
+# Loading Authorization Keys
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+In an account with multiple associated keys (multisig account), the key weights and thresholds define the hierarchy of the keys. You can extend this hierarchy using the list_authorization_keys API. This API lets you access the account hashes of the keys that sign a deploy.
```
Comment (mpapierski) [inline source/docs/casper/dapp-dev-guide/load-auth-keys.md:6]: Not sure about the existing conventions in the docs, but I'd put backticks around list_authorization_keys like this: `list_authorization_keys.` Once 1.5 is released with this API, it'd be worth replacing backticks with a link to docs.rs

**Review by mpapierski — APPROVED** [review]
Sounds good to me

**Comment by ipopescu** [comment]
Please rebase against the dev branch.

File: source/docs/casper/dapp-dev-guide/load-auth-keys.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- Authorization keys are always a subset of the associated keys of the account under which the deploy is executed. For example, if an account has two associated keys, anyone or both of the keys can be used to sign a deploy. The key that signs the deploy becomes the authorization key for that execution.
+
+## Benefits of Load Authorization Keys Feature
+The load authorization keys feature helps identify the keys that were used to sign a deploy. Let's try to understand why this feature is important for smart contract developers.
+
+- The list_authorization_keys API when called from by a smart contract, provides the set of account hashes representing all the keys used to sign the deploy under which the contract is executing. This gives developers more fine-grained control within their smart contracts.  
+- Developers can use the authorization keys to set restrictions on which keys are allowed to approve the deploy and which aren't.
+- Developers can also define the hierarchy and limit access for certain operations to specific keys.
```
Comment (Fraser999) [inline source/docs/casper/dapp-dev-guide/load-auth-keys.md:17]:
```suggestion
A smart contract can retrieve the set of authorization keys for the given execution by calling the `runtime::list_authorization_keys` function.  This returns the set of account hashes representing all the keys used to sign the deploy under which the contract is executing.

This gives developers more fine-grained control within their smart contracts.  For example, developers can define a hierarchy within an account's associated keys, and use that along with the current execution's authorization keys to limit access for certain operations to specific keys.
```

**Review by Fraser999 — COMMENTED** [review]
The function we're describing is `list_authorization_keys`, not "load", so I'd rename the two new files.

I don't think the image adds any extra info - I'd be inclined to remove it, but I don't feel strongly.  If the image is kept though, I'd suggest the following changes:
* WASM -> Wasm
* "List the account hash" -> "List the account hashes"
* "list_authorization_keys API is used to return the..." -> "list_authorization_keys returns the..."

File: source/docs/casper/dapp-dev-guide/load-auth-keys.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+
+# Accessing Authorization Keys
+The load authorization keys feature helps smart contract developers access the authorization keys of the current execution. Before the introduction of the load authorization keys feature, a smart contract couldn't find out which authorization keys were used for a given execution. Different executions of the same smart contract can have different authorization keys. Using the list_authorization_keys API, any smart contract developer can find out the keys used to sign a deploy.
```
[the remark this answers is not in the document]
Comment (GloreenS) [reply source/docs/casper/dapp-dev-guide/load-auth-keys.md:3]: Moved the paragraph to the Benefits section

File: source/docs/casper/dapp-dev-guide/load-auth-keys.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+
+# Loading Authorization Keys
```
[the remark this answers is not in the document]
Comment (GloreenS) [reply source/docs/casper/dapp-dev-guide/load-auth-keys.md]: Updated to listing

**Comment by GloreenS** [comment]
> The function we're describing is `list_authorization_keys`, not "load", so I'd rename the two new files.
> 
> I don't think the image adds any extra info - I'd be inclined to remove it, but I don't feel strongly. If the image is kept though, I'd suggest the following changes:
> 
> * WASM -> Wasm
> * "List the account hash" -> "List the account hashes"
> * "list_authorization_keys API is used to return the..." -> "list_authorization_keys returns the..."


Deleted the image as it was for the previous topic structure and was not called in the rewritten topic.

**Pull request merged**